serializer.rst

```diff
@@ -368,6 +368,83 @@ stored in one of the following locations:
 
 .. _serializer-enabling-metadata-cache:
 
+Using nested attributes
+-----------------------
+
+To map nested properties, a ``SerializedPath`` can be defined with annotations, 
```

Comment:
you need to add a `.. versionadded:: 6.2` directive, check some codebase for doc example
it helps understand that this feature is only available from 6.2+ ;)